tests/utils/utils.go

```diff
@@ -1175,3 +1179,39 @@ func GetChainConfigWithOffChainMessages(offChainMessages []avalancheWarp.Unsigne
 
 	return string(offChainMessageJson)
 }
+
+// read in the template file, make the substitutions declared at the beginning
+// of the function, write out the instantiation to a temp file, and then return
+// the path to that temp file.
+func InstantiateGenesisTemplate(
+	templateFileName string,
+	chainID uint64,
+) string {
+	substitutions := []struct {
+		Target string
+		Value  string
+	}{
+		{
+			"<EVM_CHAIN_ID>",
+			strconv.FormatUint(chainID, 10),
+		},
+	}
+
+	templateFile, err := os.Open(templateFileName)
```

Comment:
do we need to `defer templateFile.close()`? (and same for the `subnetGenesisFile`)

Reply:
Alternatively, since we're reading the whole file either way, I think we can just use `os.ReadFile` here rather than scanning it line by line below.

Reply:
good catch, thank you. in https://github.com/ava-labs/teleporter/pull/401/commits/bc9b1b6963eefde530e41c186684330cadaf51d6 I changed to `ReadFile` and also did the `defer...Close` for the `subnetGenesisFile`.